src/main/resources/db/migration/V2__update_tables.sql

```diff
@@ -92,7 +84,14 @@ ALTER TABLE IF EXISTS attachment
     MODIFY file LONGTEXT NOT NULL;
 
 
+ALTER TABLE IF EXISTS incident
+    DROP COLUMN ID;
```

Comment:
will cause issue if table exists and column id does not

Reply:
```ALTER TABLE IF EXISTS incident DROP COLUMN IF EXISTS ID;```

Would this resolve the issue?